results/prepare_data.py

```diff
@@ -693,3 +712,40 @@ def procedural_text_for_result(direction, fwb, napr_child):
     fwb.append(Paragraph(text, style))
 
     return fwb
+
+
+def previous_laboratory_result(value):
+    styleSheet = getSampleStyleSheet()
+    style = styleSheet["Normal"]
+    style.fontName = "FreeSans"
+    style.fontSize = 8
+    style.alignment = TA_JUSTIFY
+
+    opinion = [
+        [Paragraph('Анализ', style), Paragraph('Тест', style), Paragraph('Значение', style), Paragraph('Ед.изм', style),
+         Paragraph('Дата', style), Paragraph('Исполнитель', style)]
+    ]
+
+    value = json.loads(value)
+    temp_data = [[Paragraph(f"{data.get('researchTitle', '')}", style),
+                  Paragraph(f"{data.get('fractionTitle', '')}", style),
+                  Paragraph(f"{data.get('value', '')}", style),
+                  Paragraph(f"{data.get('units', '')}", style),
+                  Paragraph(f"{data.get('date', '')}", style),
+                  Paragraph(f"{short_fio_dots(data.get('docConfirm', ''))}", style)
+                  ] for data in value]
+    opinion.extend(temp_data)
+
+    tbl = Table(opinion, colWidths=(50 * mm, 35 * mm, 20 * mm, 15 * mm, 20 * mm, 30 * mm,))
+    tbl.setStyle(
+        TableStyle(
+            [
+                ('GRID', (0, 0), (-1, -1), 1.0, colors.black),
+                ('BOTTOMPADDING', (0, 0), (-1, -1), 1.5 * mm),
+            ]
+        )
+    )
+    prepare_fwb = []
+    prepare_fwb.append(tbl)
+
+    return prepare_fwb
```

Comment:
Три строчки схлопываются до 
```suggestion
    return [tbl]
```